vendor/blockchain_contracts/src/bin/calculate_offsets/ethereum/rfc003/contract.rs

```diff
@@ -0,0 +1,79 @@
+extern crate regex;
+use crate::calculate_offsets::ethereum::rfc003::{
+    compile_contract::compile, metadata::Metadata, offset::Offset,
+    placeholder_config::PlaceholderConfig, Error,
+};
+
+const TEMPLATE_FOLDER: &str = "./src/bin/calculate_offsets/ethereum/rfc003/templates/";
+const CONTRACT_FILE: &str = "contract.asm";
+const HEADER_FILE: &str = "deploy_header.asm";
+const CONFIG_FILE: &str = "config.json";
+
+pub struct Contract {
+    bytes: Vec<u8>,
+    placeholder_config: PlaceholderConfig,
+    pub meta_data: Metadata,
+}
+
+impl Contract {
+    pub fn compile_from_directory_and_load_placeholder_config(
+        dir: &'static str,
```

Comment:
🔧 as this represent a directory path, `Path` type should be more appropriate.

Reply:
it does not represent the path at this point, it is basically just the asset name (it was named asset before, but somehow that caused confusion as well).

Reply:
what about `asset_dir` or `asset_name` ?

Reply:
I changed it according to Thomas suggestions concerning the paths, see: https://github.com/comit-network/comit-rs/commit/921045499e807291a16d95aa465d63b6b8dce5a7